packages/documentation/copy/en/handbook-v2/Type Manipulation/Generics.md

```diff
@@ -426,3 +426,126 @@ A generic parameter default follows the following rules:
 - If a default type is specified and inference cannot choose a candidate, the default type is inferred.
 - A class or interface declaration that merges with an existing class or interface declaration may introduce a default for an existing type parameter.
 - A class or interface declaration that merges with an existing class or interface declaration may introduce a new type parameter as long as it specifies a default.
+
+## Variance Annotations
+
+> This is an advanced feature for solving a very specific problem, and should only be used in situations where you've identified a reason to use it
```

Comment:
Do we typically use quote blocks for emphasis?

Reply:
Sometimes, e.g. https://www.typescriptlang.org/docs/handbook/2/functions.html#function-type-expressions